### ISSUE TITLE
gx manifest.json for dev6 lists dev?

### ISSUE THREAD

**Issue by aspence** [Working-Title-MSFS-Mods/fspackages#1083]
should the manifest.json file in the gx package be updated to dev6 for the dev6 version? says dev5 i think.
thanks for this great mod.

**Comment by kaosfere** [comment]
Yes, it should.  :)   I'll make sure it gets updated in the next version we'll hopefully have out soon with a fix for the SVT issues some folks have.
